buildSrc/src/main/java/org/elasticsearch/gradle/DistributionDownloadPlugin.java

```diff
@@ -285,23 +285,52 @@ private static String distributionProjectPath(ElasticsearchDistribution distribu
         return projectPath;
     }
 
+    /**
+     * Works out the gradle project name that provides a distribution artifact. The implementation is
+     * complicated by the fact that we're inconsistent regarding where the distribution architecture
+     * appears in the name, and whether a distribution even exists for non-x86 architectures.
```

Comment:
> The implementation is complicated by the fact that we're inconsistent regarding where the distribution architecture appears in the name

We should make this consistent then. My preference would be for the convention we take with the `:archives` project with the arch being a suffix. We should rename the docker and packages subprojects to match that convention so things are consisten IMO. @jasontedor thoughts?

Reply:
I'm definitely in favour of making it consistent, but I'd prefer to do it as a separate change.

Reply:
If you like, I could the work to make the project names consistent and merge that first.

Reply:
I raised PR #55150.

Reply:
@mark-vieira This sounds good to me.